_docs-v4/custom-views/custom-view-with-js.md

```diff
@@ -45,19 +48,20 @@ class CustomView extends View {
 
 export default createPlugin({
   views: {
-    custom: CustomView
+    NameOfMyCustomViewHere: NameOfMyCustomViewHere
```

Comment:
I'd like to demonstrate that the name of the class can be one thing, and the name of the string identifier can be something else. So, it'd be great if this was `myCustomView: MyCustomView`